zookeeper/uninstalled.sls

```diff
@@ -25,16 +25,6 @@ zk-directories-removed:
 zookeeper-reload-systemctl:
   module.run:
     - name: service.systemctl_reload
-    - require:
+    - onchanges:
       - file: zk-directories-removed
 {%- endif %}
-  
-zookeeper-config-link-removed:
-  alternatives.remove:
-    - name: zookeeper-config-link
-    - path: {{ zk.real_config }}
-
-zookeeper-home-link-removed:
-  alternatives.remove:
-    - name: zookeeper-home-link
-    - path: {{ zk.real_home }}
```

Comment:
Since the `zk.alt_*` symlinks would be removed above, these states should work just fine. You may add `onlyif` parameter to check if the alternatives really exist.

Yeah, the `alternatives` state is acting like it is plain state-less execution module. That's sucks.